Header header
detection/BoundingBox[] bounding_boxes